lib/model/model.go

```diff
@@ -2018,6 +2018,7 @@ func (m *Model) CommitConfiguration(from, to config.Configuration) bool {
 	// by themselves.
 	from.Options.URAccepted = to.Options.URAccepted
 	from.Options.URUniqueID = to.Options.URUniqueID
+	from.Options.ListenAddresses = to.Options.ListenAddresses
```

Comment:
Probably. Not part of this issue though, and the deprecated stuff should never be set after parsing the config anyhow?